Formula/imagemagick.rb

```diff
@@ -72,49 +73,51 @@ def install
       --enable-static
     ]
 
-    if build.without? "modules"
-      args << "--without-modules"
-    else
-      args << "--with-modules"
-    end
-
-    if build.with? "opencl"
-      args << "--enable-opencl"
-    else
-      args << "--disable-opencl"
-    end
-
-    if build.with? "openmp"
-      args << "--enable-openmp"
-    else
-      args << "--disable-openmp"
-    end
-
-    if build.with? "webp"
-      args << "--with-webp=yes"
-    else
-      args << "--without-webp"
+    %W[
+      hdri hdri
+      opencl opencl
+      openmp openmp
+      zero-configuration zero-configuration
+    ].each_slice(2) do |option, feature|
+      if build.with? option
+        args << "--enable-#{feature}"
+      else
+        args << "--disable-#{feature}"
+      end
     end
 
-    if build.with? "openjpeg"
-      args << "--with-openjp2"
-    else
-      args << "--without-openjp2"
+    %W[
+      fftw fftw
```

Comment:
I think those guidelines are just coming from rubocop basically, which doesn't complain about %w, but you have quite some work to do if you'd like to analyze it that way:
```
Josephs-MacBook-Pro:Formula joe$ grep -r '%W' . |wc -l
     437
Josephs-MacBook-Pro:Formula joe$ grep -r '%w' . |wc -l
     313
```